hudi-utilities/src/test/java/org/apache/hudi/utilities/schema/TestSchemaRegistryProvider.java

```diff
@@ -75,7 +76,7 @@ public void testGetSourceSchemaShouldRequestSchemaWithCreds() throws IOException
     assertNotNull(actual);
     assertEquals(actual, getExpectedSchema(json));
     verify(spyUnderTest, times(1)).setAuthorizationHeader(eq(basicAuth),
-        Mockito.any(HttpURLConnection.class));
+        Mockito.any(HttpsURLConnection.class));
```

Comment:
This doesn't really test the `setUpSSLStores`. Does it?

Reply:
No. To test, I found an ssl client server example online and modified it to match the structure of schemaRegistryProvider and got that to work.